test/random/TestInverseTransformSamplingCustom.cpp

```diff
@@ -0,0 +1,254 @@
+
+// -*- C++ -*-
+/***************************************************************************
+ *
+ * The IPPL Framework
+ *
+ * This program was prepared by PSI.
+ * All rights in the program are reserved by PSI.
+ * Neither PSI nor the author(s)
+ * makes any warranty, express or implied, or assumes any liability or
+ * responsibility for the use of this software
+ *
+ * Visit www.amas.web.psi for more details
+ *
+ ***************************************************************************/
```

Comment:
I would remove this. @srikrrish ?

Reply:
Yes I told the same. I thought they were already removed (I didn't point out each and every file but mentioned these should be removed in all the files)